exploration-module/src/main/java/io/edpn/backend/exploration/adapter/persistence/entity/MybatisBodyEntity.java

```diff
@@ -0,0 +1,60 @@
+package io.edpn.backend.exploration.adapter.persistence.entity;
+
+import io.edpn.backend.exploration.application.dto.persistence.entity.BodyEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EqualsAndHashCode
```

Comment:
Playing the devil's advocate here, but are setters and no-args constructors necessary, since there is a builder?

Reply:
Whats the memory cost of leaving these?
Removed because, agreed theyre not needed. Just curious.